### DISCUSSION TITLE
adhesive forces between 2 cell

### DISCUSSION THREAD

**Discussion by sinahidd3n** [UvaCsl/HemoCell#18] [General discussion]
Hello,
I am working with hemocell and I want to define adhesive forces between 2 different types of cells. could you please tell me how should I define forces between 2 different cells?
Thanks

**Comment by ghost** [comment]
You can define any arbitrary force between the surface point of cells or between the cell surface and the geometry boundaries.
Take a look at 
- "HemoCellFields::HemoRepulsionForce" for an example of external repulsive force between cells and 
- "HemoCellFields::HemoBoundaryRepulsionForce" for force between cells and boundaries.
